_posts/2024-10-16-18.0.0-release.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Apache Arrow 18.0.0 Release"
+date: "2024-10-16 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 18.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+on [**YYY distinct commits**][2] from [**ZZZ distinct contributors**][2].
+See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 17.0.0 release, JJJJJ has been invited to be committer.
+No new members have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Linux packages notes
+
+
+## C Data Interface notes
+
+
+## Arrow Flight RPC notes
+
```

Comment:
@lidavidm can you help with the Fight notes?

Reply:
```suggestion

**Flight UCX is deprecated.** We plan to remove this experiment in the next couple of releases.

The Java implementation now transparently handles compressed Arrow data when reading, instead of requiring explicit configuration. (GH-43469)

The Ruby bindings now support implementing DoPut on the server. (GH-43814)
```